fixit/cli/add_new_rule.py

```diff
@@ -64,10 +65,19 @@ def is_path_exists(path: str) -> Path:
         return filepath
 
 
-def create_rule_file(file_path: Path) -> None:
+def is_valid_name(name: str) -> str:
+    """Check for valid rule name """
+    if name.endswith(("Rule", "Rules", "rule", "rules")):
+        raise NameError("Please enter rule name without the suffix, `rule` or `Rule`")
+    return snake_to_camelcase(name) if name else ""
```

Comment:
You can omit `if name else ""` since `snake_to_camelcase` with empty string would result with an empty string.

Reply:
done